diff --git a/node_modules/@sveltejs/kit/src/exports/vite/index.js b/node_modules/@sveltejs/kit/src/exports/vite/index.js
index 028d873..1e43886 100644
--- a/node_modules/@sveltejs/kit/src/exports/vite/index.js
+++ b/node_modules/@sveltejs/kit/src/exports/vite/index.js
@@ -797,8 +797,8 @@ function kit({ svelte_config }) {
 					);
 
 					// avoid making the manifest available to users
-					fs.unlinkSync(`${out}/client/${vite_config.build.manifest}`);
-					fs.unlinkSync(`${out}/server/${vite_config.build.manifest}`);
+					//fs.unlinkSync(`${out}/client/${vite_config.build.manifest}`);
+					//fs.unlinkSync(`${out}/server/${vite_config.build.manifest}`);
 
 					if (kit.adapter) {
 						const { adapt } = await import('../../core/adapt/index.js');
