diff --git a/node_modules/@sveltejs/kit/src/core/adapt/builder.js b/node_modules/@sveltejs/kit/src/core/adapt/builder.js
index c7e57bd..a278a76 100644
--- a/node_modules/@sveltejs/kit/src/core/adapt/builder.js
+++ b/node_modules/@sveltejs/kit/src/core/adapt/builder.js
@@ -183,7 +183,7 @@ export function create_builder({
 		writeClient(dest) {
 			return copy(`${config.kit.outDir}/output/client`, dest, {
 				// avoid making vite build artefacts public
-				filter: (basename) => basename !== '.vite'
+				//filter: (basename) => basename !== '.vite'
 			});
 		},
 
